tofhir-server/src/main/scala/io/tofhir/server/util/LogServiceClient.scala

```diff
@@ -27,15 +27,26 @@ class LogServiceClient(logServiceEndpoint: String) {
    *
    * @param projectId
    * @param jobId
-   * @param page
-   * @return A future of a tuple containing the details of individual executions and total number of executions
+   * @param page desired number of page
+   * @param rowPerPage row count per page
+   * @param dateBefore last date of the filtered executions
+   * @param dateAfter start date of the filtered executions
+   * @param errorStatuses desired error status of the filtered executions
```

Comment:
This param includes a list of error status separated by comma. Update the documentation, accordingly.